src/App.ts

```diff
@@ -98,6 +98,11 @@ export interface ViewConstraints {
   type?: 'view_closed' | 'view_submission';
 }
 
+export interface AuthConstraints {
```

Comment:
While other interfaces in this class are used for method arguments, `AuthConstraints` is used internally only so far. I think we still don't need to expose it by having `export`.